src/styles/_mixins.scss

```diff
@@ -16,4 +16,222 @@
     @media (min-width: $screen-desktop) {
         @content;
     };
-};
\ No newline at end of file
+};
+
+// Margin Mixins
+
+// Example 1:
+// input: @include margin()
+// renders:
+// At mobile: margin: .75rem 
+// At tablet: margin: 1rem
+// At desktop: margin: 1rem
+
+// Example 2:
+// input: @include margin(m,bottom,left)

```

Comment:
Alternative format for comments: 
const sum(a, b) => {
  // input: a and b, required, integers above 0
  // output: integer, the sum of a and b
};

Reply:
Also consider typescript

Reply:
Agreed, will switch to typescript block-style comments for future annotations in both js and scss